packages/react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -433,6 +442,17 @@ export const CalloutContentBase: React.FunctionComponent<ICalloutProps> = React.
       targetWindow,
     );
 
+    // do not set both top and bottom css props from positions
+    // instead, use maxHeight
+    const isForcedInBounds = positions?.elementPosition.top && positions.elementPosition.bottom;
```

Comment:
The second part is missing a `?` sign:

```suggestion
    const isForcedInBounds = positions?.elementPosition.top && positions?.elementPosition.bottom;
```